plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergRegisterTableProcedure.java

```diff
@@ -91,7 +92,7 @@ protected QueryRunner createQueryRunner()
         queryRunner.installPlugin(new TpchPlugin());
         queryRunner.createCatalog("tpch", "tpch");
 
-        Path dataDir = queryRunner.getCoordinator().getBaseDataDir().resolve("iceberg_data");
+        dataDir = queryRunner.getCoordinator().getBaseDataDir().resolve("iceberg_data");
```

Comment:
potential relevant for a follow-up PR: address the todo `TODO convert to IcebergQueryRunner when there is a replacement for HadoopTables that works with TrinoFileSystem`

We'd be working then with `local://....` (see e.g. TestIcebergV2)

Reply:
Created issue for that: https://github.com/trinodb/trino/issues/21134